packages/fluentui/react-northstar/src/components/Avatar/AvatarStatus.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import {
+  useFluentContext,
+  useTelemetry,
+  useStyles,
+  useAccessibility,
+  getElementType,
+  useUnhandledProps,
+  ComponentWithAs,
+} from '@fluentui/react-bindings';
+import { commonPropTypes, SizeValue, UIComponentProps, createShorthandFactory, createShorthand } from '../../utils';
+import * as customPropTypes from '@fluentui/react-proptypes';
+import * as PropTypes from 'prop-types';
+import { ShorthandValue, FluentComponentStaticProps } from '../../types';
+import { Accessibility, statusBehavior as avatarStatusBehavior, StatusBehaviorProps } from '@fluentui/accessibility';
+import { BoxProps } from '../Box/Box';
+import { AvatarStatusIcon } from './AvatarStatusIcon';
+
+export interface AvatarStatusProps extends UIComponentProps {
+  /** Accessibility behavior if overridden by the user. */
+  accessibility?: Accessibility<StatusBehaviorProps>;
+
+  /** A custom color. */
+  color?: string;
+
+  /** Shorthand for the icon, to provide customizing status */
+  icon?: ShorthandValue<BoxProps>;
+
+  /** Size multiplier */
+  size?: SizeValue;
+
+  /** The pre-defined state values which can be consumed directly. */
+  state?: 'success' | 'info' | 'warning' | 'error' | 'unknown';
+}
+
+export type AvatarStatusStylesProps = Required<Pick<AvatarStatusProps, 'color' | 'size' | 'state'>>;
+export const avatarStatusClassName = 'ui-avatar__status';
```

Comment:
Do we consider that this would be a breaking change for the `Avatar` component ?

Reply:
Looks like it